src/main/flight/failsafe.c

```diff
@@ -174,7 +174,7 @@ void failsafeOnValidDataReceived(void)
         // using the BST flag since no other suitable name....
     }
 
-    if (cmp32(failsafeState.validRxDataReceivedAt, failsafeState.validRxDataFailedAt) > (int32_t)failsafeState.rxDataRecoveryPeriod){
+    if (cmp32(failsafeState.validRxDataReceivedAt, failsafeState.validRxDataFailedAt) > (int32_t)failsafeState.receivingRxDataPeriodPreset){ // receivingRxDataPeriodPreset or rxDataRecoveryPeriod ??
```

Comment:
Should be solved before merging.

Reply:
Thanks.  I have checked.  It is best to use the `failsafeState.receivingRxDataPeriodPreset` duration.  This is the same as `rxDataRecoveryPeriod` except when a switch is used, when we want immediate return of control.
This code can likely be refactored in 4.4 since all durations are the same except in the switch situation.